sitemedia/js/components/SearchFilter.js

```diff
@@ -0,0 +1,31 @@
+import FilterChoice from './FilterChoice'
```

Comment:
Comments or docstrings would be really helpful to help understand the difference between `SearchFilter` and `SearchFacet`